File: 10 - Python/Exercícios/01 - Estrutura Sequencial/Exercício 10.py
#Exercício - Duração

duracao = int(input("Digite a duração em segundos: "))

horas = duracao // 3600
resto = duracao % 3600

minutos = resto // 60
segundos = resto % 60

print(f"Duração Total = {horas}:{minutos}:{segundos}")

File: 06 - C/Exercícios/01 - Estrutura Sequencial/Exercício 10.c
//Exercício - Duração

#include <stdio.h>

int main() {

    int duracao, horas, resto, minutos, segundos;

    printf("Digite a duracao em segundos: ");
    scanf("%d", &duracao);

    horas = duracao / 3600;
    resto = duracao % 3600;
      
    minutos = resto / 60;
    segundos = resto % 60;

    printf("Duracao Total = %d:%d:%d", horas, minutos, segundos);

    return 0;

}